css/main.css

```diff
@@ -25,5 +25,19 @@ body {
 }
 
 .dot {
-  stroke: #000;
+  stroke: gray;
```

Comment:
Probably don't want to make it gray to select. Make it normal. better UX. The user may not think that these dots are usable.